releasenotes/notes/pulse-gate-calibrations-78fd3fa5a5328761.yaml

```diff
@@ -42,3 +42,10 @@ features:
 
     Circuits with pulse gates may still be lowered to the Pulse Schedule representation by using
     the :py:mod:`qiskit.scheduler`.
+
+    The calibrated gate can also be now be transpiled using the regular transpilation process::
```

Comment:
```suggestion
    The calibrated gate can also be transpiled using the regular transpilation process::
```

Reply:
Fixed in https://github.com/Qiskit/qiskit-terra/pull/5080/commits/78bdbb09195b2df1927d2150981c806862c57ec3.